compose/utils.py

```diff
@@ -60,13 +64,21 @@ def split_buffer(stream, splitter=None, decoder=lambda a: a):
             yield item
 
     if buffered:
-        yield decoder(buffered)
+        try:
+            yield decoder(buffered)
+        except ValueError as e:
+            log.error(
+                'Compose tried parsing the following chunk as a JSON object, '
+                'but failed:\n%s' % repr(buffered)
+            )
+            raise StreamParseError(e)
```

Comment:
It's not necessary for this particular issue, but if we have another failure in this area of the code in the future, it would make it easier to debug.

We can modify the `except` clause to catch any `Exception` instead?